src/shared/Internal.Shared.Tasks/buildMultiTargeting/Internal.Shared.Tasks.props

```diff
@@ -0,0 +1,22 @@
+﻿<Project>
```

Comment:
I could just import the other file from here. That would suffice for now, but might get difficult to manage if we add more things to these files.